--- bin/md2short.sh
+++ bin/md2short.sh
@@ -89,6 +89,11 @@ else
     TEMPLATE='template.docx'
 fi

+if [[ -z "${PANDOC}" ]]; then
+    echo "PANDOC environment variable not set"
+    exit 1
+fi
+
 # Create a temporary data directory
 echo "Creating temporary directory."
 export PANDOC_DATA_DIR
@@ -103,7 +108,7 @@ echo "Files extracted."

 # Run pandoc
 echo "Running Pandoc."
-pandoc \
+"${PANDOC}" \
   "--from=$FROM_FORMAT" \
   --to=docx \
   --lua-filter="$SHUNN_SHORT_STORY_DIR/shunnshort.lua" \
