--- bin/md2short.sh
+++ bin/md2short.sh
@@ -104,6 +104,11 @@ else
     TEMPLATE='template.docx'
 fi

+if [[ -z "${PANDOC}" ]]; then
+    echo "PANDOC environment variable not set"
+    exit 1
+fi
+
 # Create a temporary data directory
 echo_if_verbose "Creating temporary directory."
 export PANDOC_DATA_DIR
@@ -118,7 +123,7 @@ echo_if_verbose "Files extracted."

 # Run pandoc
 echo_if_verbose "Running Pandoc."
-pandoc \
+"${PANDOC}" \
   "--from=$FROM_FORMAT" \
   --to=docx \
   "--metadata=shunn_verbose:$VERBOSE" \
